specs/market-api.yaml

```
@@ -641,39 +604,86 @@ paths:
           to return to the Negotiation phase by sending a new Proposal.
 
         * `Cancelled` - Indicates that the Requestor himself has called
-         `cancelAgreement`, which effectively stops the Agreement handshake.
+          `cancelAgreement`, which effectively stops the Agreement handshake.
 
       responses:
-        '200':
+        200:
           description: Agreement approval result.
           content:
             application/json:
               schema:
                 type: string
                 enum: [Approved, Rejected, Cancelled]
-        '401':
-          $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '408':
-          $ref: '#/components/responses/TimedOut'
-        '409':
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        408:
+          $ref: 'common.yaml#/responses/Timeout'
+        409:
           description: Agreement not in `Pending` state.
-        '410':
-          $ref: '#/components/responses/Expired'
+        410:
+          $ref: 'common.yaml#/responses/Gone' # Expired
         default:
-          $ref: '#/components/responses/UnexpectedError'
+          $ref: 'common.yaml#/responses/UnexpectedError'
 
+  /agreements/events:
```

we need to change it, because it is conflicting with `/agreements/{id}`
```suggestion
  /agreementEvents:
```

---

OK